plugins/modules/aci_match_route_destination.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_match_route_destination
+short_description: Manage Match action rule based on the Route Destination. (rtctrl:MatchRtDest)
+description:
+- Match action rule based on the Route Destination for Subject Profiles on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  subject_profile:
+    description:
+    - Name of an exising subject profile.
+    type: str
+    aliases: [ subject_name ]
+  ip:
+    description:
+    - The IP address of the route destination.
+    type: str
+  aggregate:
+    description:
+    - Option to enable/disable aggregated route.
+    type: str
+    choices: [ "no", "yes" ]
+  from_prefix_lenght:
```

Comment:
typo in lenght, must be length

several occurrences so probably wider replace is needed